metadata_mapper/mappers/validator.py

```diff
@@ -202,17 +202,18 @@ def validate(self, key: str, rikolti_data: dict,
 
         if not self.rikolti_data or not self.comparison_data:
             missing_data_desc = "mapped Rikolti" if not self.rikolti_data else "Solr"
-            error = {
-                "key": self.key,
-                "level": "ERROR",
-                "description": f"No {missing_data_desc} data found for key"
-            }
-            self.errors.add(**error)
-
-        self.before_validation()
+            print(f"ERROR: No {missing_data_desc} data found for key {self.key}")
+            # error = {
+            #     "key": self.key,
+            #     "level": "ERROR",
+            #     "description": f"No {missing_data_desc} data found for key"
+            # }
+            # self.errors.add(**error)
```

Comment:
Should lines 205-211 be uncommented, and line 205 deleted?

Reply:
I'm assuming Tim is continuing to work on this, so I just commented out the parts of his work that caused me to be unable to run validation reports at this precise juncture. We're currently discussing these errors (or lack thereof) for missing records in Rikolti.

Reply:
Ah OK, sounds good!